lib/generators/templates/application_adapter.js.es6

```diff
@@ -0,0 +1,11 @@
+// Override the default adapter with the `DS.ActiveModelAdapter` which
+// is built to work nicely with the ActiveModel::Serializers gem.
+export default DS.ActiveModelAdapter.extend({
+  // Adds X-CSRF-Token to all REST requests.
+  // Allows for the use of Rails protect_from_forgery
+  // The CSRF Token is normally found in app/views/layouts/application.html.*
+  // inserted with the rails helper: "csrf_meta_tags"
+  headers: {
+    "X-CSRF-Token": $('meta[name="csrf-token"]').attr('content')
```

Comment:
in ember-cli we have a config module we can import information from, maybe we should consider that?

```js
import config from 'app-name/config/environment';


```